client/src/context/InstanceCreationRoute.js

```diff
@@ -30,8 +32,10 @@ export const InstanceCreationRoute = () => {
       stringMinCredits, setMinCredits,
       stringMaxCredits, setMaxCredits,
       gradingScale, setGradingScale,
+      startingPeriod, setStartingPeriod,
+      endingPeriod, setEndingPeriod
     }
-  ), [courseType, startDate, endDate, teachers, stringMinCredits, stringMaxCredits, gradingScale, addedAttainments, suggestedAttainments, attainmentIncrementId]);
+  ), [courseType, startDate, endDate, teachers, stringMinCredits, stringMaxCredits, gradingScale, startingPeriod, endingPeriod, addedAttainments, suggestedAttainments, attainmentIncrementId]);
```

Comment:
Should we somehow separate these onto rows so the line wouldn't be so long

Reply:
I divided that into two, hope it's okay now